.github/workflows/node.js.yml

```diff
@@ -1,18 +1,15 @@
-# This workflow will install node dependencies, run tests and report coverage
-# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
-
 name: build
 
 on:
   push:
     branches:
-      - feat-test-CI-implementation-187419042
-      - "*"
+      - feat-user-login-187419055

```

Comment:
We can't always add branch here, this must be global to avoid touching a lot of files.